admin/js/boldgrid-backup-admin-settings.js

```diff
@@ -282,13 +282,31 @@ BoldGrid.Settings = function( $ ) {
 		}
 	};
 
+	self.toggleCompressionInfo = function() {
+		var isSystemZip = false,
```

Comment:
This could be compressed a bit...

```js
		var compressorSelector = $( 'select[name="compressor"]' );

		if ( 'system_zip' === $( compressorSelector ).val() ) {
			$( '.compression-level' ).show();
		} else {
			$( '.compression-level' ).hide();
		}
```

---

or...
```js
		if ( 'system_zip' === $( compressorSelector ).val() ) {
			isSystemZip = true;
		}
```

could become...
```js
		isSystemZip = 'system_zip' === $( compressorSelector ).val();
```